packages/i18nify-js/README.md

```diff
@@ -27,26 +27,32 @@ Go on, code conqueror, the adventure awaits!
 
 Welcome to the command center for your i18n experience! This module serves as the control hub, housing the essential functions to manage your i18n settings seamlessly. This module offers a trio of functions to handle all your i18n needs. Whether it's checking the current state, customizing settings, or starting afresh, this module has got you covered in managing your i18n world! 🚀
 
-#### Functions
+#### setState(newState)
 
-**setState(newState: Partial `<I18nState>`):** Customize and update your i18n state with ease! Whether you're changing locales or tweaking directions, this function is your ticket to tailor your i18n experience precisely how you want it! 🎨
+Customize and update your i18n state with ease! Whether you're changing locales or tweaking directions, this function is your ticket to tailor your i18n experience precisely how you want it! 🎨
 
-```
-import { setState } from "@razorpay/i18nify-js/core";
+##### Examples
```

Comment:
We can include 1 more API.. `getLocale` which detects locale from browser and returns it. This can be a sync utility.

Reply:
We already have this utility in `.internal`. Reusing that.